### DISCUSSION TITLE
Installation of Linuxbrew on Older Systems

### DISCUSSION THREAD

**Discussion by jwhite007** [Homebrew/discussions#2634] [Linux] [answered]
From https://docs.brew.sh/Homebrew-on-Linux is "Homebrew can install its own current versions of glibc and gcc for older distributions of Linux." I'm looking for a bit of clarification on what actually constitutes an "older distribution of Linux." I'm having a doozy of a time trying to reinstall Linuxbrew on an older RHEL, Red Hat Enterprise Linux Server release 6.7 (Santiago). Does the older distribution still have to have glibc >= 2.13? RHEL 6.7 can only go up to glibc 2.12, and I'm thinking that this is most probably responsible for the problems I'm having.

**Comment by iMichka** [comment, marked as answer]
Hi. 2.13 is the minimum version we support. With 2.12 you will be out of luck. Maybe if you are able to grab and old version of Linuxbrew, but even then, a lot of other things might be broken.

Also, we are soon going to migrate to gcc 7 for builds, probably with Ubuntu 18.04, and so we will increase the glibc requirement in 2022.

**Reply by jwhite007** [reply]
@iMichka,

Thanks for the quick reply.

**Reply by jwhite007** [reply]
@iMichka,

I suppose there are certain preliminary binaries that use the system glibc and point to shared objects such as /lib64/libm.so.6 which have to be from glibc >=2.13 for the bootstrap up to using the latest glibc provided by Linuxbrew itself. Right?

I'm guessing that it wouldn't suffice to install the latest glibc to a new location, add it to the LD_LIBRARY_PATH, and then attempt a Linuxbrew installation. Is this correct?
